collect/central_collector.go

```diff
@@ -797,6 +797,10 @@ func (c *CentralCollector) send(status *centralstore.CentralTraceStatus) {
 			sp.Data[k] = v
 		}
 
+		if c.hostname != "" && c.Config.GetAddHostMetadataToTrace() {
+			sp.Data["meta.refinery.sender.local_hostname"] = c.hostname
```

Comment:
nit: using some flavor of `host.name` conforms better to the otel host semconv: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/host.md

Reply:
ohh, so like `meta.refinery.sender.host.name` ?

Reply:
Ya I like that better